src/MOI_wrapper/MOI_wrapper.jl

```diff
@@ -3409,7 +3417,7 @@ function MOI.get(model::Optimizer, ::MOI.ListOfConstraintTypesPresent)
 end
 
 function MOI.get(model::Optimizer, ::MOI.ObjectiveFunctionType)
-    if model.is_feasibility
+    if !model.is_objective_set
```

Comment:
I think in other wrappers I removed the `return nothing`. If the objective isn't set, getting `MOI.ObjectiveFunctionType` doesn't have a valid return, so can return anything.

Reply:
Done